### DISCUSSION TITLE
Flutter project

### DISCUSSION THREAD

**Discussion by Mounirou12** [simolus3/drift#2790] [Q&A]
hello, please I have a problem executing my flutter project, I need help with these following errors
![Capture d’écran du 2023-12-14 20-16-49](https://github.com/simolus3/drift/assets/108185180/0f430c7a-71a9-45ed-baf8-41e461af2d58)

Launching lib/main.dart on SM A102U in debug mode...
/home/mounirou/AndroidStudioProjects/gestion_tache/android/app/src/debug/AndroidManifest.xml Error:
	Overlay manifest:package attribute declared at AndroidManifest.xml:2:5-41 value=(com.example.flutterappnoti)
	does not match the module's namespace (com.example.gestion_tache).
	Suggestion: remove the overlay declaration at AndroidManifest.xml.
	If you want to customize the applicationId for a buildType or productFlavor, consider setting applicationIdSuffix or using the Variant API.
	For more information, see https://developer.android.com/studio/build/build-variants

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Manifest merger failed : Overlay manifest:package attribute declared at AndroidManifest.xml:2:5-41 value=(com.example.flutterappnoti)
  	does not match the module's namespace (com.example.gestion_tache).
  	Suggestion: remove the overlay declaration at AndroidManifest.xml.
  	If you want to customize the applicationId for a buildType or productFlavor, consider setting applicationIdSuffix or using the Variant API.
  	For more information, see https://developer.android.com/studio/build/build-variants

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 23s
Exception: Gradle task assembleDebug failed with exit code 1

Exited (1).

**Comment by simolus3** [comment]
It's unlikely that this is caused by or related to drift, so I recommend asking on Stackoverflow or a community group for Flutter if you need help with your Flutter setup.

It might be that your package id in the `build.gradle` doesn't match the one from your manifests or your Kotlin/Java package names. Did you change those recently? You may have to make them consistent. But again, this forum is for drift-specific issues and I can't really help you with that problem.
